### ISSUE TITLE
AWS Flow Ruby Support

### ISSUE THREAD

**Issue by mcorner** [fredjean/simpler_workflow#7]
AWS has announced that they are releasing SWF Flow for Ruby: https://forums.aws.amazon.com/thread.jspa?threadID=122497&tstart=0

This may either remove the need for simpler_workflow all together, or make it better.  We will see what it looks like soon.

**Comment by fredjean** [comment]
Definitively something to keep an eye on. I'm curious to see what it ends up looking like. A lot of people assocated with the AWS GitHub organization seem to have been watching this repo...

**Comment by fredjean** [comment]
It looks like it has been announced and released: [https://github.com/flowdock/hubot-flowdock]. It'll be interesting to see where it goes.

**Comment by mcorner** [comment]
Indeed, I should be trying it out soon.  Something we have been contemplating is building something to cleanly integrate human workers into the work flow.  Possibly by providing a rails engine with a basic administrative console, ability to claim jobs, escalate jobs that aren't done, etc. etc.  Haven't had time to start that yet though.
